src/zcbor_decode.c

```diff
@@ -171,14 +171,36 @@ static bool value_extract(zcbor_state_t *state,
 	return true;
 }
 
+static int val_to_int(zcbor_major_type_t major_type, void *result, size_t result_size)
+{
+#ifdef ZCBOR_BIG_ENDIAN
```

Comment:
Do we have users of BIG_ENDIAN? when is it needed?

Reply:
I don't know if there are users, but it's not a big burden to maintain.